src/Nordea/Components/DropdownFilter.elm

```diff
@@ -117,7 +116,7 @@ init { onInput, input, onSelect, items } =
         , onSelect = onSelect
         , input = input
         , onFocus = Nothing
-        , hasFocus = True
+        , hasFocus = False
```

Comment:
Should double check that this doesn't break anything in FF. I think the partnerhub super user login page in FF may rely on this

Reply:
Nice spot! It seemed not necessary to have this set to `True`. I'll check it out

Reply:
It seems to only be in use in the configuration page, and for the PartnerHub page, both seem to work fine with it being `False` 😊